evcache-client/src/main/java/com/netflix/evcache/config/Archaius1PropertyRepo.java

```diff
@@ -0,0 +1,209 @@
+package com.netflix.evcache.config;
+
+import java.util.Optional;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+import com.netflix.config.ChainedDynamicProperty;
+import com.netflix.config.DynamicIntProperty;
+import com.netflix.config.DynamicLongProperty;
+import com.netflix.config.DynamicPropertyFactory;
+import com.netflix.config.DynamicStringProperty;
+import com.netflix.config.DynamicStringSetProperty;
+import com.netflix.config.Property;
+import com.netflix.config.PropertyWrapper;
+import com.netflix.config.ChainedDynamicProperty.ChainLink;
+
+/**
+ * Archaius 1 implementation of PropertyRepo with com.netflix.config.** 
+ * 
+ */
+public class Archaius1PropertyRepo implements PropertyRepo {
+	static class DynamicPropertyAdapter<T> implements Prop<T> {
+		private Property<T> property;
+
+		DynamicPropertyAdapter(Property<T> property) {
+			this.property = property;
+		}
+
+		@Override
+		public T get() {
+			if  (property instanceof ChainLink) {
```

Comment:
Can/should the instanceof check be done in the constructor?

Reply:
removed, not necessary now that ChainedPropertyAdapter exists